react-delegationdashboard/src/components/Overview/Cards/index.tsx

```diff
@@ -138,6 +139,18 @@ const Views = () => {
           <AutomaticActivationAction automaticFlag={contractOverview.automaticActivation} />
         </StatCard>
       )}
+      {isAdmin() && location.pathname === '/owner' && (
+        <StatCard
+          title="ReDelegate Cap"
+          value={contractOverview.reDelegationCap === 'true' ? 'ON' : 'OFF'}
+          color="green"
+          svg="activation.svg"
+          percentage="Cap for rewards"
+          tooltipText="If your agency uses a max delegation cap and the ReDelegate Cap is OFF your delegators will be able to redelegate the reward to your agency. If the value is ON then the redelegate wil not be accepted."
```

Comment:
Can you fix the typo (If the value is ON then the redelegate will not be accepted.)